### ISSUE TITLE
Documentation website: Insecure connection

### ISSUE THREAD

**Issue by JayAndCatchFire** [ReactTraining/react-router#6517]
The documentation website's https certificate appears to be misconfigured.  Firefox no longer allows me to view the site.  I only noticed this happening this week.  Last week, it was fine.  I don't know if it's because something about the certificate changed or if this is a new issue after my upgrade from Firefox 63 to 64.

![screenshot from 2018-12-18 19-29-29](https://user-images.githubusercontent.com/10983817/50193230-5eadb500-02fb-11e9-8dfe-e54c50500b28.png)

Here's what the "learn more" link points to:
https://support.mozilla.org/en-US/kb/what-does-your-connection-is-not-secure-mean

If I click on the "SEC_ERROR_UNKNOWN_ISSUER", here's what I get:

```
https://reacttraining.com/react-router/

Peer’s Certificate issuer is not recognized.

HTTP Strict Transport Security: true
HTTP Public Key Pinning: false

Certificate chain:

-----BEGIN CERTIFICATE-----
MIIDLTCCApagAwIBAgIHFUFXZgHlajANBgkqhkiG9w0BAQsFADCBpzELMAkGA1UE
BhMCVFcxDzANBgNVBAgMBlRhaXdhbjEPMA0GA1UEBwwGVGFpcGVpMRYwFAYDVQQK
DA1TeW5vbG9neSBJbmMuMR4wHAYDVQQLDBVDZXJ0aWZpY2F0ZSBBdXRob3JpdHkx
GTAXBgNVBAMMEFN5bm9sb2d5IEluYy4gQ0ExIzAhBgkqhkiG9w0BCQEWFHByb2R1
Y3RAc3lub2xvZ3kuY29tMB4XDTE4MTEwNzA3NDMyMVoXDTM4MDcyNTA3NDMyMVow
gZYxCzAJBgNVBAYTAlRXMQ8wDQYDVQQIDAZUYWl3YW4xDzANBgNVBAcMBlRhaXBl
aTEWMBQGA1UECgwNU3lub2xvZ3kgSW5jLjERMA8GA1UECwwIRlRQIFRlYW0xFTAT
BgNVBAMMDHN5bm9sb2d5LmNvbTEjMCEGCSqGSIb3DQEJARYUcHJvZHVjdEBzeW5v
bG9neS5jb20wgZ8wDQYJKoZIhvcNAQEBBQADgY0AMIGJAoGBANLY2e1VM6F+bevg
a2+juCLgZusPPurV6TDZLrOaH2BQuUd/73xMk9alF6jOdJ0Bi7lilaSvv56hsDlJ
ZoqJ9QwYXCwZEskbYnFHdK2CH5Wjoy0wnzouulGkkHmvqFcOlCYtWmBnBrPlXGE4
BXFQtXCheRVdX+9LYw36S5EdUulLAgMBAAGjcjBwMB8GA1UdEQQYMBaBFHByb2R1
Y3RAc3lub2xvZ3kuY29tMDoGCWCGSAGG+EIBDQQtFittb2Rfc3NsIGdlbmVyYXRl
ZCBjdXN0b20gc2VydmVyIGNlcnRpZmljYXRlMBEGCWCGSAGG+EIBAQQEAwIGQDAN
BgkqhkiG9w0BAQsFAAOBgQASzDB2v8gY7Igv9ggPA8Cr18fk62qLG5qm+x5uv1sW
wNXOJc+nsxnCB0d8GUOkAPB1i6TfIw3IWIerYoBMLuEixNHjip3jyjyYaHcKFF4+
t24wrpv7TySEZMkDROXYW68smBN0r6PiXXAnSWVr3dw06Ar1X1UQNIqo6dJWI1X3
Vw==
-----END CERTIFICATE-----
```

**Comment by JayAndCatchFire** [comment]
I just realized that you probably want to know what the "Advanced" tab contains.

![screenshot from 2018-12-18 19-29-29](https://user-images.githubusercontent.com/10983817/50193230-5eadb500-02fb-11e9-8dfe-e54c50500b28.png)

**Comment by JayAndCatchFire** [comment]
If I click on the "SEC_ERROR_UNKNOWN_ISSUER", here's what I get:

```
https://reacttraining.com/react-router/

Peer’s Certificate issuer is not recognized.

HTTP Strict Transport Security: true
HTTP Public Key Pinning: false

Certificate chain:

-----BEGIN CERTIFICATE-----
MIIDLTCCApagAwIBAgIHFUFXZgHlajANBgkqhkiG9w0BAQsFADCBpzELMAkGA1UE
BhMCVFcxDzANBgNVBAgMBlRhaXdhbjEPMA0GA1UEBwwGVGFpcGVpMRYwFAYDVQQK
DA1TeW5vbG9neSBJbmMuMR4wHAYDVQQLDBVDZXJ0aWZpY2F0ZSBBdXRob3JpdHkx
GTAXBgNVBAMMEFN5bm9sb2d5IEluYy4gQ0ExIzAhBgkqhkiG9w0BCQEWFHByb2R1
Y3RAc3lub2xvZ3kuY29tMB4XDTE4MTEwNzA3NDMyMVoXDTM4MDcyNTA3NDMyMVow
gZYxCzAJBgNVBAYTAlRXMQ8wDQYDVQQIDAZUYWl3YW4xDzANBgNVBAcMBlRhaXBl
aTEWMBQGA1UECgwNU3lub2xvZ3kgSW5jLjERMA8GA1UECwwIRlRQIFRlYW0xFTAT
BgNVBAMMDHN5bm9sb2d5LmNvbTEjMCEGCSqGSIb3DQEJARYUcHJvZHVjdEBzeW5v
bG9neS5jb20wgZ8wDQYJKoZIhvcNAQEBBQADgY0AMIGJAoGBANLY2e1VM6F+bevg
a2+juCLgZusPPurV6TDZLrOaH2BQuUd/73xMk9alF6jOdJ0Bi7lilaSvv56hsDlJ
ZoqJ9QwYXCwZEskbYnFHdK2CH5Wjoy0wnzouulGkkHmvqFcOlCYtWmBnBrPlXGE4
BXFQtXCheRVdX+9LYw36S5EdUulLAgMBAAGjcjBwMB8GA1UdEQQYMBaBFHByb2R1
Y3RAc3lub2xvZ3kuY29tMDoGCWCGSAGG+EIBDQQtFittb2Rfc3NsIGdlbmVyYXRl
ZCBjdXN0b20gc2VydmVyIGNlcnRpZmljYXRlMBEGCWCGSAGG+EIBAQQEAwIGQDAN
BgkqhkiG9w0BAQsFAAOBgQASzDB2v8gY7Igv9ggPA8Cr18fk62qLG5qm+x5uv1sW
wNXOJc+nsxnCB0d8GUOkAPB1i6TfIw3IWIerYoBMLuEixNHjip3jyjyYaHcKFF4+
t24wrpv7TySEZMkDROXYW68smBN0r6PiXXAnSWVr3dw06Ar1X1UQNIqo6dJWI1X3
Vw==
-----END CERTIFICATE-----
```

**Comment by timdorr** [comment]
The cert's valid on my end and was just recently issued via Let's Encrypt. You could have caught it during a reissue.

**Comment by JayAndCatchFire** [comment]
I just tried again, but this time on my phone using Firefox for Android and I'm still getting the equivalent error for mobile. 

![screenshot_20181219-014018](https://user-images.githubusercontent.com/10983817/50205993-b108c900-032f-11e9-8647-2d0565cbb6b6.png)

**Comment by JayAndCatchFire** [comment]
I take it back. I just switched off of my Wi-Fi and onto my mobile network and it seems to work now. It's probably a caching issue on my end.

**Comment by JayAndCatchFire** [comment]
Okay, I figured out what my issue was.  Here's what's up in case anyone else ever runs into this issue.

I have a Synology router and one of the recent firmware updates added this new "safe browsing" thingy that uses Google's safe browsing system/rules that they use in Chrome.  Looks like my definitions were out of date.  For whatever reason, that caused a whole bunch of sites, including the React Router and React Training sites, to be blocked.  Looks like it was a subset of sites that use Let's Encrypt.

I updated the definitions manually and everything seems to work now.  Weird, eh?
